build/nl_server/Dockerfile

```diff
@@ -31,6 +31,13 @@ RUN pip3 install --no-cache-dir -r /workspace/nl_server/requirements.txt
 COPY nl_server/. /workspace/nl_server/
 COPY shared/. /workspace/shared/
 
+# Download nl files from gcs
+COPY deploy/nl/catalog.yaml .
+COPY build/nl_server/requirements.txt /workspace/build/nl_server/requirements.txt
+COPY build/nl_server/download_nl_files.py .
+RUN pip3 install -r /workspace/build/nl_server/requirements.txt
+RUN python3 download_nl_files.py
```

Comment:
wonder how much larger is the new docker image?

Reply:
about 300MB larger: https://screenshot.googleplex.com/9k9jurE7fquXmKe